docs/source/compute_config/aws_lambda.md

```diff
@@ -44,34 +43,72 @@ python3 -m pip install lithops[aws]
 
 7. Choose **Lambda** on the use case list and click **Next: Permissions**. Select the policy created before (`lithops-policy`). Click **Next: Tags** and **Next: Review**. Type a role name, for example `lithops-execution-role`. Click on *Create Role*.
 
-## Configuration
+## AWS Credential setup
 
-6. Edit your lithops config and add the following keys:
+Lithops loads AWS credentials as specified in the [boto3 configuration guide](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html).
 
+In summary, you can use the following settings:
+
+1. Provide credentials via the `~/.aws/config` file. **This is the preferred option to configure AWS credentials for use with Lithops**:
+
+    You can run `aws configure` command if the AWS CLI is installed to setup the credentials.
```

Comment:
How the `~/.aws/config` looks like in this case? are the keys going into a default profile by defaut? or are the keys set in the file without a profile? 

I mean, after calling `aws configure`, you get this:?

```
aws_access_key_id=XXXXXXXXX
aws_secret_access_key=XXXXXX
```
or something like this:
```
[profile default]
aws_access_key_id=XXXXXXXXXXXXXXX
aws_secret_access_key=XXXXXXXXXXXXXXXX
```

Reply:
I wonder if in this case it makes sense to force the user to provide a `profile_name` with `aws configure --profile my-unique-profile-name` and then confiure lithops like in the SSO approach, with:

```
lithops:
    backend: aws_lambda

aws:
    config_profile: my-unique-profile-name

aws_lambda:
    execution_role: <EXECUTION_ROLE_ARN>
    region: <REGION_NAME>
```